rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -167,8 +167,17 @@ size_t
 NodeGraph::count_publishers(const std::string & topic_name) const
 {
   auto rcl_node_handle = node_base_->get_rcl_node_handle();
+  auto name = rcl_node_get_name(rcl_node_handle);
+  auto namespace_ = rcl_node_get_namespace(rcl_node_handle);
```

Comment:
please use trailing underscores only for member variables. same below.

Reply:
@Karsten1987 I have'nt read variables with '_' suffix, is that a valid declaration within "ros2" coding styles? May I use it here?

Reply:
We normally adhere to the C++ coding guidelines: https://google.github.io/styleguide/cppguide.html#Variable_Names

Now, I think you can actually use it here, because namespace is a reserved keyword in C++. I've seen we used it in other places as well, so that I believe in this special case, it's fine.
Please ignore my comment then.